src/operation/delete/test.rs

```diff
@@ -180,6 +180,7 @@ async fn handle_write_concern_failure() {
                 code: 456,
                 code_name: "wcError".to_string(),
                 message: "some message".to_string(),
+                err_info: None,
```

Comment:
Instead of having `err_info` be `None` here, it would be useful to add an `errInfo` field to the `CommandResponse` document above and make sure it's properly included in the error -- ditto for the other tests below.

Reply:
yep makes sense. fixed